src/experiment/trials/videos.js

```diff
@@ -1,30 +1,45 @@
+import store from "store2";
 import jsPsychVideoKeyboardResponse from "@jspsych/plugin-video-keyboard-response";
 import jsPsychImageButtonResponse from "@jspsych/plugin-image-button-response";
-import { config, jsPsych } from "./config";
-import { imgContent, videoContent } from "./preload";
+// import { imgContent, mediaAssets.video, mediaAssets } from "../../preload";
+import { generateAssetObject } from "@bdelab/roar-utils";
+import assetsFile from '../../../assets.json';
+
+const bucketURI = 'https://storage.googleapis.com/roar-mep';
+const mediaAssets = generateAssetObject(assetsFile, bucketURI);
 
 const isMobile = () => /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
 console.log(`isMobile: ${isMobile()}`);
```

Comment:
roar-utils has a isMobile() function now.

Reply:
I ran npm update, but it seems like isMobile still isn't being recognized as a function in @bdelab/roar-utils

Reply:
My bad, it's actually getDevice() which returns a string of 'mobile' or 'desktop'.